### ISSUE TITLE
When no history is present there, narrator does not announce “There’s no history yet”

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#1384]
A team member will update this issue with more details soon.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.
